src/univariate/continuous/chernoff.jl

```diff
@@ -137,11 +138,11 @@ module ChernoffComputations
 
         (y > 0) || throw(DomainError(y, "argument must be positive"))
 
-        cnsty = y^(-1.5)
+        cnsty = inv(sqrt(y^3))
         if (y <= 1.0)
-            return sum([(b[k]*cnsty - a[k]*sqrthalfpi)*y^(3*k) for k=1:length(a)])-sqrthalfpi
+            return sum((b[k]*cnsty - a[k]*sqrthalfpi)*y^(3.0*k) for k in 1:length(a)) - sqrthalfpi
         else
-            return sum([exp(cuberoottwo*a*y) for a in airyai_roots]) * 2 * sqrttwopi * exp(-y*y*y/6) - cnsty
+            return sum(exp(cuberoottwo*a*y) for a in airyai_roots) * 2.0 * sqrttwopi * exp(-y^3 / 6.0) - cnsty
```

Comment:
```suggestion
            return sum(exp(cuberoottwo*a*y) for a in airyai_roots) * 2 * sqrttwopi * exp(-y^3 / 6) - cnsty
```

Reply:
since the results of the calculations are float, it may be ok to avoid promoting?

Reply:
While the results will be floating point numbers it is not clear that they are of type `Float64`. There are many issues and PRs that fix type instability issues introduced by Float64 literals, so it is almost always better to avoid them.